.python_package/src/gridtools/__init__.py

```diff
@@ -0,0 +1,20 @@
+# GridTools
```

Comment:
I have the feeling that `gridtools` as package/module name is a bit ambiguous (some people could think that `gridtools` means `gt4py`). Have you thought about using `gridtools_cpp` instead, which would also match the package distribution name?